src/user/model.coffee

```diff
@@ -11,9 +11,12 @@ BLANK_USER =
   name: null
   profile_url: null
   courses: []
+  _course_data: []
+  endpoints: {}
+  isLoaded: false
```

Comment:
Hm, I think we can't set the endpoints to blank here.  If we do so then once the user logouts out it'll be reset to an empty object (since we're now `_.extend(this, BLANK_USER)` below.

If we lose the `endpoints.login` property then the login gateway component won't have anywhere to open.